src/lib.rs

```diff
@@ -59,6 +60,33 @@ pub enum DisplayState {
     Editable,
 }
 
+/// Configure spacing and separators for the view
+#[derive(Debug, Clone)]
+pub struct HexViewConfig {
+    /// Number of bytes per line
```

Comment:
Controls how many bytes per line will be displayed. 

    Default is 16

Question for you: What does happen on 0?

Reply:
The same as if bytes_per_line is lower than bytes_per_group, curses doesn't even start. Do you think we should document it or do something else, like adding a check somewhere? Maybe even if just to add a more helpful error message.

Reply:
A panic is probably fine, when trying to display the view. If we want to make an error able way, we could add a `validate` method on the `HexViewConfig` struct and check that on set_config or on the view method itself.